client/package.json

```diff
@@ -108,34 +109,33 @@
     "@types/react": "^18.3.3",
     "@types/react-autosuggest": "^10.1.5",
     "@types/react-dom": "^18.3.0",
-    "@types/react-highlight-words": "^0.16.4",
+    "@types/react-highlight-words": "^0.20.0",
     "@types/react-router-hash-link": "^2.4.5",
     "@types/react-test-renderer": "^18.0.0",
     "@types/recompose": "^0.30.7",
     "@types/redux-mock-store": "^1.0.3",
     "@types/sortablejs": "^1.13.0",
     "@types/styled-components": "^5.1.1",
-    "@types/uuid": "^9.0.1",
+    "@types/uuid": "^10.0.0",
     "@types/webrtc": "^0.0.43",
-    "@typescript-eslint/eslint-plugin": "^5.10.1",
-    "@typescript-eslint/parser": "^5.10.1",
+    "@typescript-eslint/eslint-plugin": "^7.16.0",
+    "@typescript-eslint/parser": "^7.16.0",
     "csvtojson": "^2.0.10",
     "eslint": "^8.28.0",
-    "eslint-config-next": "^14.2.4",
+    "eslint-config-next": "^14.2.5",
     "jest": "^29.3.1",
     "jest-environment-jsdom": "^29.3.1",
     "jest-mock-axios": "^4.7.0-beta2",
     "jest-styled-components": "^7.0.0",
-    "next-router-mock": "^0.8.0",
+    "next-router-mock": "^0.9.13",
     "papaparse": "^5.3.0",
     "postcss": "^8.4.5",
     "react-test-renderer": "^18.2.0",
     "redux-mock-store": "^1.5.4",
     "redux-saga-test-plan": "^4.0.0-rc.3",
     "sass": "^1.57.1",
-    "ts-jest": "29.0.3",
-    "typescript": "^4.9.3",
-    "typescript-plugin-css-modules": "^5.1.0"
```

Comment:
has it been removed because it's not working anymore or for another reason?

Reply:
Yes I was falling victim to https://github.com/mrmckeb/typescript-plugin-css-modules/issues/221 but I managed to get things working.  It requires setting up a `paths` configuration for scss files in `tsconfig.json` and updating the `@import` statements in .scss files.